azurerm/resource_arm_relay_namespace.go

```diff
@@ -173,11 +193,21 @@ func resourceArmRelayNamespaceRead(d *schema.ResourceData, meta interface{}) err
 		d.Set("location", azureRMNormalizeLocation(*location))
 	}
 
-	if sku := resp.Sku; sku != nil {
-		flattenedSku := flattenRelayNamespaceSku(sku)
-		if err = d.Set("sku", flattenedSku); err != nil {
-			return fmt.Errorf("Error setting `sku`: %+v", err)
+	if skuName == "" {
```

Comment:
As above, we can simply set both

Reply:
Keeping setting of both, but grouped the logic so it's easier to read...